cmd/cri-containerd/cri_containerd.go

```diff
@@ -164,6 +164,39 @@ func startProfilingServer(host string, port string) {
 	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
 	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
 	if err := http.ListenAndServe(endpoint, mux); err != nil {
-		glog.Errorf("Failed to start profiling server: %v", err)
+		logrus.WithError(err).Error("Failed to start profiling server")
+	}
+}
+
+func setLogLevel(l string) error {
+	lvl, err := log.ParseLevel(l)
+	if err != nil {
+		return err
+	}
+	if err := setGLogLevel(lvl); err != nil {
+		return err
+	}
+	logrus.SetLevel(lvl)
+	return nil
+}
+
+// TODO(random-liu): Set glog level in plugin mode.
+func setGLogLevel(l logrus.Level) error {
+	if err := flag.Set("logtostderr", "true"); err != nil {
+		return err
+	}
+	switch l {
+	case log.TraceLevel:
+		return flag.Set("v", "5")
+	case logrus.DebugLevel:
+		return flag.Set("v", "4")
+	case logrus.InfoLevel:
+		return flag.Set("v", "2")
+	// glog doesn't support following filters.
+	case logrus.WarnLevel:
+	case logrus.ErrorLevel:
+	case logrus.FatalLevel:
+	case logrus.PanicLevel:
```

Comment:
maybe "0" for these?

Reply:
Set "0", means V(0) will still be printed, which is not what we want.